imap/jmap_calendar.c

```diff
@@ -4092,11 +4092,17 @@ static int jmap_calendarevent_get(struct jmap_req *req)
 
     if (hashu64_count(&rock.cache_jsevents)) {
         r = caldav_begin(db);
-        if (!r) hashu64_enumerate(&rock.cache_jsevents,
+        if (!r) {
+            hashu64_enumerate(&rock.cache_jsevents,
                 cachecalendarevents_cb, &rock);
-        if (r) caldav_abort(db);
-        else r = caldav_commit(db);
-        if (r) goto done;
+            r = caldav_commit(db);
+        }
```

Comment:
Is there no case in which we want to call caldav_abort()?

Reply:
No.

Reply:
At least none that I could see to be relevant. We could plumb any errors from the callback up, though.